install.sh

```diff
@@ -96,7 +98,16 @@ mkdir -p $TMPDIR/bin/
 echo "[PRE-INSTALL] [HELM] Checking HELM installation..."
 echo "[PRE-INSTALL] [HELM]: Downloading Helm $HELM_VERSION"
 curl --fail -L ${HELM_URL} | tar zxf - --directory="$TMPDIR/bin/" --wildcards '*/helm' --strip 1
-git clone "$URL" $TMPDIR/liqo --depth 1
+if [ "$LIQO_SUFFIX" == "-ci" ]; then
+  git clone "$URL" $TMPDIR/liqo
+  cd  $TMPDIR/liqo
+  git pull origin
+  git checkout "$LIQO_VERSION" > /dev/null 2> /dev/null
+  cd -
+else
+  git clone "$URL" $TMPDIR/liqo --depth 1
+fi
```

Comment:
A couple of comments here (if I got your goal correctly)
1. I would replace this block with something like `git clone "${URL} "${TMPDIR}/liqo" --branch "${LIQO_VERSION}"  --depth 1`, that is much simpler to understand (and it is a oneliner)
2. The variable LIQO_VERSION may not be defined here (you are setting it at line 134), so it may be better to move this down